psydac/api/ast/parser.py

```diff
@@ -1358,6 +1358,9 @@ def _visit_TensorMax(self, expr, **kwargs):
 
         return tuple(newargs)
 
+    def _visit_TensorInteger(self, expr, **kwargs):
+        args = [expr.args[0]]*self.dim
+        return tuple(args)
```

Comment:
Minor comment: I don't see a reason to create a list first, and then return a tuple 😉 

```suggestion
        return (expr.args[0],) * self.dim
```

Reply:
fixed